insonmnia/hub/server.go

```diff
@@ -1118,10 +967,40 @@ func (h *Hub) Serve() error {
 		}()
 	}
 
+	h.cluster.RegisterEntity("tasks", h.tasks)
+	h.eventCh = h.cluster.Run()
+	go h.listenClusterEvents()
+
 	h.wg.Wait()
 	return nil
 }
 
+func (h *Hub) listenClusterEvents() {
+	for {
+		select {
+		case event := <-h.eventCh:
+			h.processClusterEvent(event)
+		case <-h.ctx.Done():
+			return
+		}
+	}
+}
+
+func (h *Hub) processClusterEvent(value interface{}) {
+	log.G(h.ctx).Info("received cluster event", zap.Any("event", value))
+	switch value.(type) {
```

Comment:
switch value := value.(type)

in that case, you can omit type case in case *map[string]*TaskInfo